docs/platform/howto/cleanup-powered-off-services.rst

```diff
@@ -0,0 +1,22 @@
+Periodic cleanup of powered-off services
+========================================
+
+Aiven platform provides a way to power-off services stop accumulating usage costs when not in use. However leaving them powered-off for an extended period of time comes with disadvantages too.
+
+Aiven platform receive regular maintenance updates to keep the services updated with the latest supported versions upstream, including fixes to bugs, possible security vulnerabilities, and overall improvements to performance and memory consumption.
```

Comment:
1. The Aiven platform receives
2. Remove "possible" (all the other list items are also "possible" :-) )